content/swift/concepts/arrays/arrays.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Arrays'
-Description: 'Arrays are ordered collections of values. In other words, an array is an ordered list of values, all of the same type, contained in the same structure. Swift, being a very type safe language, will throw compiler errors if you attempt to store values of different types, say attempting to store a String in an int array. The same value can be stored in an array multiple times in different locations. Arrays can be created several ways, each with a specific use case. An empty array can be declared and initalized with the following syntax: pseudo var arrayName = Type '
+Description: 'Arrays are ordered collections of values. In other words, an array is an ordered list of values, all of the same type, contained in the same structure. Swift, being a very type safe language, will throw compiler errors if you attempt to store values of different types, say attempting to store a String in an int array. The same value can be stored in an array multiple times in different locations. Arrays can be created several ways, each with a specific use case. An empty array can be declared and initialized with the following syntax: pseudo var arrayName = Type '
```

Comment:
```suggestion
Description: 'Arrays are ordered collections of values. In other words, an array is an ordered list of values, all of the same type, contained in the same structure.'
```

Reply:
@garanews @SSwiniarski I am resolving these swift entries since I am currently working on edits to all of swift